src/main/java/ddingdong/ddingdongBE/common/exception/CustomExceptionHandler.java

```diff
@@ -125,4 +149,22 @@ private String createLogConnectionInfo(HttpServletRequest request) {
     private void loggingApplicationError(String applicationLog) {
         log.warn("errorLog = {}", applicationLog);
     }
+
+    public String getRequestBody(HttpServletRequest request) {
```

Comment:
```suggestion
    private String getRequestBody(HttpServletRequest request) {
```
접근 제어자 수정해야 할 것 같습니다!